addon/ng2/utilities/environment-plugin.ts

```diff
@@ -0,0 +1,85 @@
+import * as fs from 'fs';
+
+interface WebpackPlugin {
+  apply(compiler: any): void;
+}
+
+interface NgCliEnvrionmentConfig {
```

Comment:
1. Typo
2. This isn't used. What's the point of this interface? It's not even exported.